### ISSUE TITLE
Roadmap for Jellyseerr

### ISSUE THREAD

**Issue by gauthier-th** [Fallenbagel/jellyseerr#926]
Hi everyone,

As requested, here is a quick overview of what we're planning as next features on Jellyseerr, in approximate order:

- #868
- #685 
- Make sure that all the refactors we've done work properly (like Next.js update or the refactor to the Fetch API)
- #907 
- #632 
- #883 
- SSO: will depend on #883. We'd like to start afresh from this PR instead of merging #184 
- #628 
- Custom rules (like changing quality profile, root folder, etc.) depending on conditions (like user, genre, studio, etc.). This will supersede #757.

No ETA here, we're all volunteers and we do our best in our spare time.

This does not mean, however, that a PR that is not mentioned here will not be merged! Feel free to make new PRs 😉 

We'll try to keep this updated.

**Comment by thinhlegolas** [comment]
Hi there was a roadmap for lidarr support. Has that been killed off already?

Thank you for the hard work and wonderful job as always.

**Comment by gauthier-th** [comment]
> Hi there was a roadmap for lidarr support. Has that been killed off already?
> 
> Thank you for the hard work and wonderful job as always.

IIRC, Lidarr was added to a far too ambitious roadmap a couple years ago.
It's not abandoned, our attention is just elsewhere for the moment. It should eventually be integrated into Jellyseerr.

**Comment by Phunky** [comment]
Hopefully [this](https://github.com/sct/overseerr/pull/3800) may help with the Lidarr support 🤞

**Comment by hellez** [comment]
Is it possible to get #644 released?

**Comment by Fallenbagel** [comment]
> Is it possible to get #644 released?

its already merged? it will be released in the next version.

**Comment by devanteweary** [comment]
Love the roadmap. 
Thank you for putting it together!

Is dual Jellyfin / Plex functionality still in the works? 
I don't see it on the list but remember reading that was something being worked on.

Just curious!

**Comment by inform880** [comment]
Really looking forward to #96! Thank you contribs! Everyone in my family loves this!

**Comment by ayykamp** [comment]
Hey, I saw that Overseerr development picked back up again recently. Does this mean both projects will merge back together at some point?

**Comment by M0NsTeRRR** [comment]
> Hey, I saw that Overseerr development picked back up again recently. Does this mean both projects will merge back together at some point?

It seems so! :)

**Comment by gshpychka** [comment]
@M0NsTeRRR could we have a tracking issue for what's still left to do to complete the transition to Seerr?

**Comment by M0NsTeRRR** [comment]
> [@M0NsTeRRR](https://github.com/M0NsTeRRR) could we have a tracking issue for what's still left to do to complete the transition to Seerr?

We need at least :
- https://github.com/seerr-team/seerr/pull/2069
- https://github.com/seerr-team/seerr/pull/2019
- our new logo

After that, we need to discuss when we will release the first version, but most of the work has already been completed.

**Comment by gshpychka** [comment]
> We need at least :
> 
> * [docs: add migration guide #2069](https://github.com/seerr-team/seerr/pull/2069)
> * [feat: Overseerr to Jellyseerr migration #2019](https://github.com/seerr-team/seerr/pull/2019)
> * our new logo
> 
> After that, we need to discuss when we will release the first version, but most of the work has already been completed.

Thanks. You mentioned #2019 that's already merged - is there any work left there?

For the logo - what's the task there? To create the new logo, or to switch the old one to the new one everywhere?

What about little things like renaming `docs/using-jellyseerr`, updating all the links in e.g. the README to point to the seerr domain, updating the Weblate listing (and all the translations), and switching all the other leftover references [throughout the repo](https://github.com/search?q=repo%3Aseerr-team%2Fseerr%20%22jellyseerr%22&type=code) (e.g. `git clone https://github.com/fallenbagel/jellyseerr.git` in the "build from source" instructions)?

Might be helpful to have a comprehensive tracking issue to keep all of these little tasks in one place.

**Comment by gauthier-th** [comment]
> Thanks. You mentioned [#2019](https://github.com/seerr-team/seerr/pull/2019) that's already merged - is there any work left there?

No, it's now done.

> For the logo - what's the task there? To create the new logo, or to switch the old one to the new one everywhere?

We have to do both, a new logo and change it everywhere.

> What about little things

We are currently working on it (see #2111)

**Comment by kaaax0815** [comment]
I think it should be noted on the https://seerr.dev/ site that this migration is happening.
As new users will be confused as screenshots and docs differ and stuff like that

on a side note, what is the recommended way to get started with seerr now? use jellyseerr and then migrate to seerr? wait for the unified version?

**Comment by gauthier-th** [comment]
> I think it should be noted on the https://seerr.dev/ site that this migration is happening.

The Seerr website is just a nice overview of the app. The README of the GitHub states that the migration is happening, and the docs still points to the Jellyseerr docs, so it's fine like this.

> on a side note, what is the recommended way to get started with seerr now? use jellyseerr and then migrate to seerr? wait for the unified version?

Both are possible, it depends on individual preferences.

**Comment by kaaax0815** [comment]
> The Seerr website is just a nice overview of the app. The README of the GitHub states that the migration is happening, and the docs still points to the Jellyseerr docs, so it's fine like this.

I agree, I just noticed it and thought it'll probably confuse users

> Both are possible, it depends on individual preferences.

is there any estimate when the migration is in a stable / usable form?

**Comment by fallenbagel** [comment]
> > The Seerr website is just a nice overview of the app. The README of the GitHub states that the migration is happening, and the docs still points to the Jellyseerr docs, so it's fine like this.
> 
> I agree, I just noticed it and thought it'll probably confuse users
> 

Without that docs, there is no way for jellyseere users to install jellyseerr as seerr is not out yet. That's why docs are kept in it's legacy form.

> > Both are possible, it depends on individual preferences.
> 
> is there any estimate when the migration is in a stable / usable form?
> 

Seerr has been technically in a stable/usable form for a while. Though only available as develop image for now as we are currently in the process of developing a logo for it.

**Comment by headscarf737** [comment]
Any update on the logo?

**Comment by nphalem** [comment]
> Any update on the logo?

They have decided to keep the current logo. They announced it on their discord server.

@fallenbagel please replace anime specific sonarr/radarr support PR with https://github.com/seerr-team/seerr/pull/2452 inside the first message.

**Comment by ilevine8** [comment]
Any possibility of adding support for book requests down the line? I just got Chaptarr up and running which seems like a promising successor to Readarr. 

There is a Jellyseerr fork that has a book preview tag which appears to works with Chaptarr, but that is going to be left behind pretty quickly considering how fast this project is moving. It would be great to have that capability supported long term. 

Thanks everyone for the amazing work on this!

**Comment by gauthier-th** [comment]
> Any possibility of adding support for book requests down the line?

As you mentioned, the project is moving forward, and we already have a lot of features we’d like to implement (like OIDC or music support, to name just a few). Given that, support for books is clearly not a priority at the moment, even though we’re not opposed to it.

**Comment by adrianipopescu** [comment]
> > Any possibility of adding support for book requests down the line?
> 
> As you mentioned, the project is moving forward, and we already have a lot of features we’d like to implement (like OIDC or music support, to name just a few). Given that, support for books is clearly not a priority at the moment, even though we’re not opposed to it.

Is there an architectural possibility to expose an sdk to build plugins? that way the community can build at their own rate.

**Comment by gauthier-th** [comment]
> > > Any possibility of adding support for book requests down the line?
> > 
> > 
> > As you mentioned, the project is moving forward, and we already have a lot of features we’d like to implement (like OIDC or music support, to name just a few). Given that, support for books is clearly not a priority at the moment, even though we’re not opposed to it.
> 
> Is there an architectural possibility to expose an sdk to build plugins? that way the community can build at their own rate.

That's not really possible. The codebase can't be changed like that since everything is "hardcoded" for movies and series.

**Comment by fallenbagel** [comment]
> Any possibility of adding support for book requests down the line? I just got Chaptarr up and running which seems like a promising successor to Readarr. 
> 
> There is a Jellyseerr fork that has a book preview tag which appears to works with Chaptarr, but that is going to be left behind pretty quickly considering how fast this project is moving. It would be great to have that capability supported long term. 
> 
> Thanks everyone for the amazing work on this! 

Also pretty sure the fork you speak of is this one right?
https://github.com/seerr-team/seerr/pull/1918

**Comment by ilevine8** [comment]
Yes, that is what I was referring to. Appreciate your response! Understood that there are a lot of things to come first. Hopefully sometime in the future that can start to be considered. Again, really appreciate all the hard work on this.

**Comment by M0NsTeRRR** [comment]
Hello everyone,

I've updated the roadmap message. We now track upcoming features and other work in progress through milestones (see the link in the first message).

Since we're a team of volunteers, it's difficult to provide exact timelines except for the next release.

To answer a common question:
> Will OIDC, Music, and other features like that be added with that change ?

**Yes, they’re planned but we don’t have an ETA.**
